firstvoices/backend/search/utils/get_index_documents.py

```diff
@@ -19,8 +19,9 @@ def create_story_index_document(instance, page_text, page_translation):
         page_text=page_text,
         page_translation=page_translation,
         has_audio=instance.related_audio.exists(),
-        has_video=instance.related_videos.exists(),
+        has_document=instance.related_documents.exists(),
         has_image=instance.related_images.exists(),
```

Comment:
Oops, I don't think this function is used anywhere. Looks like I forgot to remove it when I converted these util methods to the IndexManagers. Could you please delete this function?